packages/auth-sdk/user.tsx

```diff
@@ -16,16 +17,46 @@ const userKeys = {
    user: ['user'],
 };
 
+const hasLocalStorage = typeof localStorage !== 'undefined';
+const userDataLocalKey = 'user.data';
+
 export function useAuthenticatedUser() {
    const apiClient = useIFixitApiClient();
+   const cachedUserData = useMemo(
```

Comment:
React also offers a `useCallback` hook that might fit this case better

Reply:
I think *callback* might be a bit disingenuous here... Though it seems like I can pass a function to `placeholderData` and that makes the code a bit cleaner. I'll do it.

https://react-query-v3.tanstack.com/guides/placeholder-query-data#placeholder-data-as-a-function

Reply:
Oh gotcha, the use of `useMemo` in the docs is interesting. Seems like it would work either way